src/disemvowling.js

```diff
@@ -0,0 +1,8 @@
+export default function disemvoweling(string) {
+  
+  var strToLowerCase = string.toLowerCase()
+  var stringWithoutSpecialChar = strToLowerCase.replace(/\W+/g, '')
+  var stringWithoutVowels = stringWithoutSpecialChar.replace(/[aeiou]/ig,'')
+
```

Comment:
Could the above three lines been all condensed into one line? (Not sure if it would actually work)

return string.toLowerCase().replace(/\W+/g, '').replace(/[aeiou]/ig,'')